{
  "title": "CVE-2022-36126",
  "cve": "CVE-2022-36126",
  "notes": {
    "description": "An issue was discovered in Inductive Automation Ignition before 7.9.20 and 8.x before 8.1.17. The ScriptInvoke function allows remote attackers to execute arbitrary code by supplying a Python script.",
    "published": "2022-07-16",
    "modified": "2022-07-16"
  },
  "references": [
    {
      "url": "https://github.com/sourceincite/randy",
      "description": "MISC:https://github.com/sourceincite/randy"
    },
    {
      "url": "https://srcincite.io/advisories/src-2022-0014/",
      "description": "MISC:https://srcincite.io/advisories/src-2022-0014/"
    },
    {
      "url": "https://support.inductiveautomation.com/hc/en-us/articles/7625759776653",
      "description": "MISC:https://support.inductiveautomation.com/hc/en-us/articles/7625759776653"
    }
  ]
}
